jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalGroupService.java

```diff
@@ -130,7 +134,48 @@ public void saveGroup(Group group) {
      * {@inheritDoc}
      */
     @Override
+    public void saveGroup(GroupAdministrationDto dto) throws NotFoundException {
+        throwValidationExceptionIfNotUnique(dto);
+        Group group = dto.getId() != null ? dao.get(dto.getId()) : new Group();
+        if (group == null) {
+            throw new NotFoundException();
+        }
+        if (!isGroupEditable(group.getName())) {
+            throw new OperationIsNotAllowed("Group is not editable");
```

Comment:
Could you please:

- Log this event as a WARN
- Specify (both to the exception and to the logs) which group we were trying to edit